pkg/uwb-core/dpl/dpl_callout.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_uwb_core
+ * @{
+ *
+ * @file
+ * @brief       uwb-core DPL (Decawave Porting Layer) callout
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ * @}
+ */
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <signal.h>
+#include <time.h>
```

Comment:
I don't think all those includes are needed here. But `#include "xtimer.h"` is missing since it's used in this file.

Reply:
its included in `dpl_callout.h`